korman/korlib/texture.py

```diff
@@ -186,3 +186,41 @@ def store_in_mipmap(self, mipmap, data, compression):
         func = mipmap.CompressImage if compression == plBitmap.kDirectXCompression else mipmap.setLevel
         for i, level in enumerate(data):
             func(i, level)
+
+
+    @staticmethod
+    def create_bump_LUT(mipmap):
+        kLUTHeight = 16
+        kLUTWidth = 16
+
+        buf = bytearray(kLUTHeight * kLUTWidth * 4)
+
+        denom = kLUTWidth - 1
+        delH = (kLUTHeight - 1) // 5
+        startH = delH // 2 + 1
+        doneH = 0
+
+        doneH = startH * kLUTWidth * 4
+        buf[0:doneH] = [b for x in range(kLUTWidth) for b in [0, 0, int((x / denom) * 255.9), 255]] * startH
```

Comment:
Tried that, but can't use generators here because we're multiplying by an int for repeat :(